webpack/dev.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+
+const baseConfig = require('./base.config.js');
+
+module.exports = merge(baseConfig, {
+  devtool: 'inline-source-map',
+  devServer: {
+    contentBase: './ui-src',
+    historyApiFallback: true,
+    inline: true,
+    port: 3000,
+    hot: true,
+    clientLogLevel: 'info',
+  },
```

Comment:
Is this a usable config without that proxy block? When I asked about making the proxy url configurable, I was hoping to allow using the config without having to modify the checked-in file. I don't think this does that; it doesn't do any proxying now, and the Javascript doesn't have a mechanism for changing the /api/ path it uses.

Hmm, this seems to work:

'/api': process.env.MOONFIRE_URL || 'http://localhost:8080',

When I run "yarn script", it uses http://localhost:8080, so it works with a local moonfire-nvr with default arguments. When I run "MOONFIRE_URL=http://odroid:8080 yarn script", I see all the data from my odroid-based setup.